pkg/virt-operator/util/config.go

```diff
@@ -431,7 +431,9 @@ func (c *KubeVirtDeploymentConfig) generateInstallStrategyID() {
 	// reason: sha1 is not used for encryption but for creating a hash value
 	hasher := sha1.New()
 	values := getStringFromFields(*c)
-	hasher.Write([]byte(values))
+	if _, err := hasher.Write([]byte(values)); err != nil {
+		panic("Error with sha1 is not possible")
```

Comment:
Are we sure we can just panic here ?  Maybe we need to change the function signature, return and error , and change the caller to check the error and return the proper values further (error, nil, etc..)

Reply:
We can add the return but I don't think that error is possible here. (I am not saying we should keep the panic, I actually used `panic` so we can discuss the places which I couldn't decide what to do)